### DISCUSSION TITLE
me getting banned from psych engine be like

### DISCUSSION THREAD

**Discussion by ErenKaya31** [YoshiCrafter29/YoshiCrafterEngine#101] [General]
https://user-images.githubusercontent.com/95748211/158868995-7ff4d83c-d06c-4d0e-9878-a93de1239fc1.mp4

**Comment by ErenKaya31** [comment]
https://user-images.githubusercontent.com/95748211/158996682-cc32e6f4-6dbf-4868-aa2e-25c953fc7964.mp4

lol

**Comment by xmati-swag** [comment]
death :skull:
